packages/edit-site/src/components/sync-state-with-url/use-sync-path-with-url.js

```diff
@@ -36,6 +36,12 @@ export function getPathFromURL( urlParams ) {
 	return path;
 }
 
+function isSubset( subset, superset ) {
```

Comment:
Just curious: this function, on the surface, appears to check equality. Is it possible to add a note to explain the relationship between `subset` and `superset`?

Or just in this comment box for me 🤣

Reply:
If all of the key/value pairs of A are also present in B then A is a subset of B and B is a superset of A.

https://en.wikipedia.org/wiki/Subset

Maybe I should call it `includes` which is what lodash used? Would that be more familiar?

https://lodash.com/docs/4.17.15#includes

Reply:
> Maybe I should call it includes which is what lodash used? Would that be more familiar?

I was only thinking of the casual observer, especially since this file's code is, or at least has been, at best, difficult to grok at first glance. 

I'm fine with the function description personally. Thanks for the explainer! 🍺